src/avtk/clipselector.cxx

```diff
@@ -90,6 +91,34 @@ void ClipSelector::clipName(int clip, std::string name)
 	redraw();
 }
 
+void ClipSelector::setClipBeats(int scene, int beats, bool isBeatsToRecord)
+{
+	clips[scene].setBeats(beats, isBeatsToRecord);
+	redraw();
+}
+
+double getCairoTextWith(cairo_t * cr, const char * str)
```

Comment:
I would like to add these non-member-functions to a namespace to prevent conflicts in the future.

Reply:
just make them "static" and that way the C++ compiler won't make them visible from outside the compilation unit, aka .cpp file

Reply:
which is literally the same like using a namespace, but its an anonymous one, right?